### ISSUE TITLE
Error: There is no formatter for 'shellscript' files installed.

### ISSUE THREAD

**Issue by panchalkalpesh** [foxundermoon/vs-shell-format#295]
### Issue: 

This extension does not work on **Visual Studio Code**:
```
Version: 1.65.2 (Universal)
Commit: c722ca6c7eed3d7987c0d5c3df5c45f6b15e77d1
Date: 2022-03-10T14:33:49.188Z
Electron: 13.5.2
Chromium: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Darwin x64 21.4.0
```

### Screen Capture: 

https://user-images.githubusercontent.com/1808686/161860451-69215f3c-17f9-4ff3-9ccf-1566b9d05387.mov

**Comment by iankkhoros** [comment]
I have the same issue. I have installed this extension along with its dependencies, golang and shfmt but I keep getting this error, despite relaunching. 
![Screen Shot 2022-05-11 at 4 51 04 PM](https://user-images.githubusercontent.com/96147165/167945631-a26a469a-911f-4f45-a721-6728bea199e9.png)

**Comment by alanlivio** [comment]
I have the same problem. VSCode suggests other extensions and the mkhl.shfmt seems to be working. 

```
> (Get-CimInstance Win32_OperatingSystem).version
10.0.22631
> wsl.exe --version | Select -First 1
WSL version: 1.3.15.0
> code.exe --version | Select -First 1
> code.exe --show-versions --list-extensions | grep latex
James-Yu.latex-workshop@9.10.0
```

**Comment by panchalkalpesh** [comment]
@foxundermoon I see you tagged this Issue with "need more information" label. 
Let us know what more information is required?

**Comment by alanlivio** [comment]
Hi, again all 👋
My status today for this issue is:
- it works on a folder opened from Windows
- it works on a folder opened from WSL
- it does not work when opening multiple folders from a FILE.code-workspace

@foxundermoon, maybe the vs-shell-format uses some path from the current folder?

**Comment by haeraeus** [comment]
Same issue using vscode remote on fresh "ubuntu-2404-noble-amd64-v20250130".

**Comment by masterwishx** [comment]
have same issue after last update

**Comment by z80020100** [comment]
https://github.com/foxundermoon/vs-shell-format/issues/396

**Comment by epignatelli** [comment]
Downgrading to 7.2.2 works as a temporary workaround, while we wait for a fix.

**Comment by italocjs** [comment]
+1, same issue.

**Comment by ottosch** [comment]
> [#396](https://github.com/foxundermoon/vs-shell-format/issues/396)

this fixed for me on 7.2.8.
FYI the directory was `~/.vscode-oss/extensions/foxundermoon.shell-format-7.2.8-universal/dist`

**Comment by wtj** [comment]
+1, same issue with 7.2.8, downgrade to 7.2.2 as a workaround.

**Comment by samueldyoung29ctr** [comment]
Can confirm that on Windows 11 (VSCode 1.103.2, using Remote-SSH to a Linux machine), version 7.2.8 does not work but downgrading to 7.2.5 works for me.

**Comment by masterwishx** [comment]
> downgrading to 7.2.5 works for me.

Yep more info here https://github.com/foxundermoon/vs-shell-format/issues/396

**Comment by ottosch** [comment]
Hard to understand why you people prefer to downgrade instead of applying the fix.

**Comment by epignatelli** [comment]
@ottosch Cause the fix is harder to port and automate for distribution, while the newer version brings very little benefits.

**Comment by damianagape** [comment]
Yes, downgrade helps. Tip: it's also possible to specify version in `.devcontainer/devcontainer.json`

```
{
  "name": "...",
  "image": "...",
  "customizations": {
    "vscode": {
      "extensions": [
        "foxundermoon.shell-format@7.2.5"
      ],
      "settings": {
        "shellformat.path": "/usr/local/bin/shfmt"
      }
    }
  }
}
```

**Comment by rj-au** [comment]
> Hard to understand why you people prefer to downgrade instead of applying the fix.

Are you referring to switching to [lumirelle/vs-shell-format](https://github.com/lumirelle/vs-shell-format)?

**Comment by ottosch** [comment]
> > Hard to understand why you people prefer to downgrade instead of applying the fix.
> 
> Are you referring to switching to [lumirelle/vs-shell-format](https://github.com/lumirelle/vs-shell-format)?

No, to simply replace the one_ini_bg node module: https://github.com/foxundermoon/vs-shell-format/issues/295#issuecomment-3162151309

**Comment by jgrumboe** [comment]
> Are you referring to switching to [lumirelle/vs-shell-format](https://github.com/lumirelle/vs-shell-format)?

Thanks for this mention, I switched, and I'm happy to use an active fork.
